src/main/kotlin/com/apurebase/kgraphql/schema/execution/ArgumentTransformer.kt

```diff
@@ -66,4 +68,36 @@ open class ArgumentTransformer(val schema : DefaultSchema) {
     fun transformPropertyValue(parameter: InputValue<*>, value: String, variables: Variables): Any? {
         return transformValue(parameter.type, value, variables)
     }
+
+    fun transformPropertyObjectValue(parameter: InputValue<*>, value: List<*>): Any? {
+        val mapper = ObjectMapper()
+        mapper.registerModule(JavaTimeModule())
```

Comment:
What's the reasoning behind adding this module?

Reply:
This module is neaded for the support of Java 8 date/time API in the FasterXML framework.
In my project i used the "java.time.LocalDate" and i got this exception : `"com.fasterxml.jackson.databind.exc.InvalidDefinitionException:  Cannot construct instance of java.time.LocalDate`
The solution was adding this module

Reply:
Wouldn't this work instead:
```kotlin
fun transformPropertyObjectValue(parameter: InputValue<*>, value: List<*>): Any? {
    return schema.configuration.objectMapper.readValue(value.toJson(), parameter.type.unwrapped().kClass?.java)
}
```

And then you would create a custom scalar for the `java.time.LocalDate` type?

Reply:
Worked perfectly 👍 
I will modify the code and commit it right away 😃